tests/latent_heat_enthalpy/statistics

```diff
@@ -0,0 +1,15 @@
+# 1: Time step number
+# 2: Time (seconds)
+# 3: Time step size (seconds)
+# 4: Number of mesh cells
+# 5: Number of Stokes degrees of freedom
+# 6: Number of temperature degrees of freedom
+# 7: Iterations for temperature solver
+# 8: Iterations for Stokes solver
+# 9: Velocity iterations in Stokes preconditioner
+# 10: Schur complement iterations in Stokes preconditioner
+# 11: Minimal temperature (K)
+# 12: Average temperature (K)
+# 13: Maximal temperature (K)
+0 0.000000000000e+00 0.000000000000e+00 256 2467 1089  0 31 32 32 9.99000000e+02 9.99500000e+02 1.00000000e+03 
+1 1.000000000000e+15 1.000000000000e+15 256 2467 1089 17 28 29 29 9.98956206e+02 1.00253772e+03 1.01739649e+03 
```

Comment:
The original test output was 9.99032569e+02 1.00157844e+03 1.01243949e+03 (for the temperature at the end time, from your old branch). So we should check why this is so different, and what's going on.

Reply:
As the `damage_rheology_latent_heat` works, could you try changing the geometry and refinement level to the same numbers used in `damage_rheology_latent_heat`, and compare the results. Based on the description of the test, I think they should be the same.